drivers/flash/spi_nor.c

```diff
@@ -1365,12 +1419,18 @@ static int spi_nor_pm_control(const struct device *dev, enum pm_device_action ac
 		break;
 #else
 	case PM_DEVICE_ACTION_SUSPEND:
-		acquire_device(dev);
+		rc = acquire_device(dev);
+		if (rc < 0) {
+			return rc;
+		}
 		rc = enter_dpd(dev);
 		release_device(dev);
 		break;
 	case PM_DEVICE_ACTION_RESUME:
-		acquire_device(dev);
+		ret = acquire_device(dev);
```

Comment:
```suggestion
		rc = acquire_device(dev);
```
There's also one more call to `acquire_device()` below: https://github.com/zephyrproject-rtos/zephyr/blob/0b132d9eb7a60c77bdda80e147f3e07c6e96f47c/drivers/flash/spi_nor.c#L1446 Why is the return value ignored there?

Reply:
Because I missed it :) fixed now